itest/lnd_multi-hop_test.go

```diff
@@ -1921,11 +1924,22 @@ func runMultiHopHtlcAggregation(ht *lntest.HarnessTest,
 	// Once bob has force closed, we can restart carol.
 	require.NoError(ht, restartCarol())
 
-	// Mine a block to confirm the closing transaction.
-	ht.MineBlocksAndAssertNumTxes(1, 1)
+	// Mine an empty block first to trigger the anchor cpfp transaction of
+	// bob.
+	ht.MineEmptyBlocks(1)
 
-	// The above mined block will trigger Bob to sweep his anchor output.
-	ht.Miner.AssertNumTxsInMempool(1)
+	// Mine a block to confirm the closing transaction and the anchor cpfp
+	// transaction.
+	ht.MineBlocksAndAssertNumTxes(1, 2)
+
+	// Trigger the acutal sweeps.
+	ht.MineEmptyBlocks(1)
+
+	// The above sweep will trigger 2 transactions
+	// The timeout transactions of bob.
+	// And the remote spend + Anchor of Carol
+	// The anchor of bob is not swept because its cltv delta is zero.
+	ht.Miner.AssertNumTxsInMempool(2)
```

Comment:
yes including the parent feerate to the fee bump logic, broke quite a lot of tests, but because I am building it back, no itest changes are required. However I want to point out that these tests are currently very hard to maintain.